migrations/20180725202700-populate-collective-with-opencollective-bot.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const Promise = require('bluebird');
```

Comment:
Any reason not to simply rely on the native Promise?

Reply:
I agree with you, but I have seen bluebird in some files of the project(but for the migration folder it's there most of the times) so I was wondering if we have a standard for it or today we actually use both bluebird and native promises. Anyways, the native one will do the job here so I'll remove.

Reply:
Yes, let's skip bluebird for new code unless there is good reasons to use it.

Reply:
Really useful if you need `Promise.map` or `Promise.props`

Reply:
If I was unclear, I generally prefer that we stick to native Promises. If we need one of the feature that are only available with Bluebird, we can just import it where needed.